internal/mode/static/state/resolver/resolver.go

```diff
@@ -52,15 +58,15 @@ func (e *ServiceResolverImpl) Resolve(ctx context.Context, svc *v1.Service, port
 	err := e.client.List(
 		ctx,
 		&endpointSliceList,
-		client.MatchingFields{index.KubernetesServiceNameIndexField: svc.Name},
-		client.InNamespace(svc.Namespace),
+		client.MatchingFields{index.KubernetesServiceNameIndexField: svcNsName.Name},
+		client.InNamespace(svcNsName.Namespace),
 	)
 
 	if err != nil || len(endpointSliceList.Items) == 0 {
-		return nil, fmt.Errorf("no endpoints found for Service %s", client.ObjectKeyFromObject(svc))
+		return nil, fmt.Errorf("no endpoints found for Service %s", svcNsName)
```

Comment:
I think this string formatting will work since `types.NamespacedName` has a `String()` method, but you might want to double-check that it doesn't print like `types.NamespacedName{Name: "name", Namespace: "ns"}`. It should be `"name/ns"`

Reply:
Checked this by printing the error message in the test file and here is what I got: 

`no endpoints found for Service test/svc`

However, the other returned error in `resolveEndpoints` here:

(svcPort is of type v1.ServicePort)
`fmt.Errorf("no valid endpoints found for Service %s and port %+v", svcNsName, svcPort)`

which I did not touch prints this:

`no valid endpoints found for Service test/svc and port {Name:http-svc-port Protocol:TCP AppProtocol:<nil> Port:80 TargetPort:{Type:0 IntVal:8080 StrVal:} NodePort:0}` 

Is that alright?

Reply:
Hmm, I think when we wrote that error message, we wanted it to include the entire port struct, but it seems like overkill to me now. I would change it to just the port number. 

@pleshakov thoughts?

Reply:
>  I would change it to just the port number.
👍